browser/src/control/Control.UIManager.js

```diff
@@ -376,6 +384,9 @@ L.Control.UIManager = L.Control.extend({
 			// remove unused elements
 			L.DomUtil.remove(L.DomUtil.get('presentation-controls-wrapper'));
 			document.getElementById('selectbackground').parentNode.removeChild(document.getElementById('selectbackground'));
+
+			if (this.getHighlightMode()) $('#colrowhighlight').addClass('selected');
+			else $('#colrowhighlight').removeClass('selected');
```

Comment:
@eszkadev Here i planned to remember toggle state for the case on notebookbar
enable highlight
refresh the document
expecting to see button toggled on
actually toggled off.

What is wrong here?

Reply:
`map['stateChangeHandler'].getItemValue('YOUR COMMAND')` should get the correct value from preferences saved in the browser.

because toolbutton use it to fetch the state. command must match what you have defined for the button's `command` in JSON.

see: https://github.com/CollaboraOnline/online/blob/25dfda07cf5414923501e51b3027f422fd2dc315/browser/src/control/Control.JSDialogBuilder.js#L2343